packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -254,6 +255,41 @@ impl DatabaseApi<'_> {
             .update_entity(entity_id, entity, entity_type_uri, self.account_id)
             .await
     }
+
+    pub async fn get_entity_links(&self, source_entity_id: EntityId) -> Result<Links, QueryError> {
+        self.store.get_entity_links(source_entity_id).await
+    }
+
+    pub async fn get_link_target(
+        &self,
+        source_entity_id: EntityId,
+        link_type_uri: VersionedUri,
+    ) -> Result<OutgoingLink, QueryError> {
+        self.store
+            .get_link_target(source_entity_id, link_type_uri)
+            .await
+    }
+
+    async fn create_link(
+        &mut self,
+        source_entity: EntityId,
+        target_entity: EntityId,
+        link_type_uri: VersionedUri,
+    ) -> Result<Link, InsertionError> {
+        let link_id = Link::new(source_entity, target_entity, link_type_uri);
+
+        self.store.create_link(link_id, self.account_id).await
```

Comment:
I think this is more a `link` rather than a `link_id`?

Reply:
Yes, sorry. Refactored the names a couple times and missed some spots.

70ea1703a54d703fa3c8fda05867833da3802482

Reply:
No worries 🙂